src/scrapers/main.py

```diff
@@ -31,22 +36,34 @@ def docs():
         print('{: <16}{}'.format(s.code, s.name))
 
 
-(opts, args) = parser.parse_args()
+arguments = parser.parse_args()
 
-if opts.help:
+if arguments.help:
```

Comment:
Werkt dit wel? Ik dacht dat argparser by default een bind dest doet op het eerste argument, dus dat zou `arguments.h` zijn in dit geval. Als je dus al je 'primaire namen' vooraan zet, en daarna de shorthand, dan werkt het out of the box;
```
parser.add_argument("--nogeo", "-n", help="Disables the geocoder", action="store_true")
```

Reply:
Ik snap niet wat je bedoelt. Het werkt prima, of ik nou -h of --help gebruik.